docs/install.rst

```diff
@@ -22,38 +22,32 @@ PlasmaPy requires a version of |Python| between |minpython| and
 |maxpython|. If you do not have |Python| installed already, here are the
 instructions to `download Python`_ and install it. 🐍
 
-.. tip::
-
-   New versions of |Python| are released annually in October, and it can
-   take a few months for the scientific Python ecosystem to catch up. If
-   you have trouble installing `plasmapy` on the most recent |Python|
-   version between October and March, then try installing it on the
-   second most recent version.
-
```

Comment:
This was primarily an issue for back when PlasmaPy depended on Numba, which often took several months to become compatible with each new version of Python. Since PlasmaPy no longer depends on Numba, the delay will be significantly shorter.